### ISSUE TITLE
Test sequence lead to motor not moving

### ISSUE THREAD

**Issue by gin66** [gin66/FastAccelStepper#25]
Start test sequence `t M1 02 R`. While motor is running, issue `02`

Result is, that motor is stuck in RED state. Issuance of another `02` resumes function:
```
[00:42:22:431] Select stepper 1
[00:42:22:433] Select test_seq_02
[00:42:22:434] Run tests
[00:42:22:435] M1: @0 M2: @0 M3: @0 M4: @0 M5: @0 M6: @0 
[00:42:22:523] M1: @1 => 0 QueueEnd=0/22400us MANU M2: @0 M3: @0 M4: @0 M5: @0 M6: @0 
[00:42:22:623] M1: @1 => 0 QueueEnd=0/22400us MANU M2: @0 M3: @0 M4: @0 M5: @0 M6: @0 
[00:42:22:723] M1: @4 => 0 QueueEnd=3/11200us ACC   M2: @0 M3: @0 M4: @0 M5: @0 M6: @0 
[00:42:22:823] M1: @3 => 7 QueueEnd=4/11200us RED   M2: @0 M3: @0 M4: @0 M5: @0 M6: @0 
[00:42:22:923] M1: @3 => 0 QueueEnd=3/11200us RED   M2: @0 M3: @0 M4: @0 M5: @0 M6: @0 
[00:42:23:023] M1: @4 => 10 QueueEnd=5/9984us RED   M2: @0 M3: @0 M4: @0 M5: @0 M6: @0 
[00:42:23:123] M1: @10 => 0 QueueEnd=9/15808us ACC   M2: @0 M3: @0 M4: @0 M5: @0 M6: @0 
[00:42:23:223] M1: @1 => 0 QueueEnd=1/15808us RED   M2: @0 M3: @0 M4: @0 M5: @0 M6: @0 
[00:42:23:323] M1: @8 => 13 QueueEnd=9/9984us RED   M2: @0 M3: @0 M4: @0 M5: @0 M6: @0 
[00:42:23:423] M1: @11 => 0 QueueEnd=9/9088us ACC   M2: @0 M3: @0 M4: @0 M5: @0 M6: @0 
[00:42:23:523] M1: @1 => 0 QueueEnd=1/15808us RED   M2: @0 M3: @0 M4: @0 M5: @0 M6: @0 
[00:42:23:623] M1: @9 => 17 QueueEnd=10/7904us RED   M2: @0 M3: @0 M4: @0 M5: @0 M6: @0 
[00:42:23:723] M1: @17 => 0 QueueEnd=17/0us ACC   M2: @0 M3: @0 M4: @0 M5: @0 M6: @0 
[00:42:23:770] Select test_seq_02
[00:42:23:771] Run tests
[00:42:23:823] M1: @6 => 1 QueueEnd=5/11200us RED   M2: @0 M3: @0 M4: @0 M5: @0 M6: @0 
[00:42:23:923] M1: @1 => 0 QueueEnd=1/0us RED   M2: @0 M3: @0 M4: @0 M5: @0 M6: @0 
[00:42:24:023] M1: @1 => 0 QueueEnd=1/0us RED   M2: @0 M3: @0 M4: @0 M5: @0 M6: @0 
[00:42:24:123] M1: @1 => 0 QueueEnd=1/0us RED   M2: @0 M3: @0 M4: @0 M5: @0 M6: @0 
[00:42:24:223] M1: @1 => 0 QueueEnd=1/0us RED   M2: @0 M3: @0 M4: @0 M5: @0 M6: @0 
[00:42:24:323] M1: @1 => 0 QueueEnd=1/0us RED   M2: @0 M3: @0 M4: @0 M5: @0 M6: @0 
[00:42:24:423] M1: @1 => 0 QueueEnd=1/0us RED   M2: @0 M3: @0 M4: @0 M5: @0 M6: @0 
[00:42:24:523] M1: @1 => 0 QueueEnd=1/0us RED   M2: @0 M3: @0 M4: @0 M5: @0 M6: @0 
[00:42:24:623] M1: @1 => 0 QueueEnd=1/0us RED   M2: @0 M3: @0 M4: @0 M5: @0 M6: @0 
[00:42:24:723] M1: @1 => 0 QueueEnd=1/0us RED   M2: @0 M3: @0 M4: @0 M5: @0 M6: @0 

```

**Comment by gin66** [comment]
Simple test on avr after reset: M1 A1000 V30 R1000 W R1
100% reproducible

**Comment by gin66** [comment]
M1 A1000 V30 R1000 W R2
works as expected....

Same behavior on esp32

**Comment by gin66** [comment]
Fixed in 0.16.5
